chat.delta.desktop.yml

```diff
@@ -0,0 +1,126 @@
+app-id: chat.delta.desktop
+tags:
+  - delta.chat
+  - email
+  - chat
+  - messenger
+  - crypto
+base: io.atom.electron.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/run.sh
+#rename-icon: deltachat-desktop
+#rename-desktop-file: deltachat-desktop.desktop
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  # We can record and play voice messages
+  - --socket=pulseaudio
+  # Without DRI the app doesn't come up :-/
+  - --device=dri
+  # It connects to the network to send and receive messages
+  - --share=network
+  # I guess it wants to show notification.
+  - --talk-name=org.freedesktop.Notifications
+
+build-options:
+    env:
+        NPM_CONFIG_LOGLEVEL: info
+
+
+modules:
+  - name: cyrus-sasl2
+    # Weird that we can disable it; I thought it was a dependency. Hm.
```

Comment:
If it works without it, just remove entire section. It's not that complicated manifest this time to justify leaving it with `disabled`.

Reply:
https://github.com/deltachat/deltachat-core#user-content-installing-c-level-dependencies mentions sasl2 being a dependency. Let me re-enable it, then. It seems deltachat is shipping a custom copy of sasl2: https://github.com/deltachat/deltachat-core/tree/master/libs/cyrussasl which is probably used when it could not be detected...